eth_crowdfund_be/README.md

```diff
@@ -4,52 +4,51 @@
 
 1. Install [Python](https://www.python.org/downloads/), [Pipenv](https://docs.pipenv.org/) and [Postgres](https://www.postgresql.org/) on your machine, if you do not have them already
   * To install Pipenv on a Mac, you can run `$ brew install pipenv`
-1. You should have Python 3.8.5
+2. You should have Python 3.8.5
   ```
   $ python3 --version
   => Python 3.8.5
   ```
-1. You should have Postgres
+3. You should have Postgres
   ```
   $ which psql
   => /Applications/Postgres.app/Contents/Versions/latest/bin/psql
   ```
-1. You should have pipenv
+4. You should have pipenv
   ```
   $ pipenv --version
   => pipenv, version 2020.8.13
   ```
-1. Fork and clone down the repository
-1. Change into the directory `$ cd eth_crowdfund_be/eth_crowdfund_api`
-1. You will need to work in a virtual environment. Why? Using a virtual environment for Python projects allows us to have an isolated working copy of Python so we can work on a specific project without worrying about affecting other projects.
-  1. Within the `/eth_crowdfund_api` directory:
-    1. Run `# pipenv --three` to create the virtual environment
-    1. Run `$ pipenv shell` to activate the project virtual environment. When you are done working on the project, you should execute `$ exit` to exit the virtual environment
-    1. Run `$ pipenv install flask flask-sqlalchemy psycopg2 flask-migrate flask-script marshmallow flask-bcrypt pyjwt` to install all dependencies
-    1. Run `$ createdb eth_crowdfund_api_db` to create the app database
-    1. Run
-    ```
-    $ export FLASK_ENV=development
-    $ export JWT_SECRET_KEY=hhgaghhgsdhdhdd
-    ```
-    to set the system environment variables.
-    You should now be able to run the app:
-    1. Run
-    ```
-    $ python3 run.py
-    =>  * Serving Flask app "src.app" (lazy loading)
-        * Environment: development
-        * Debug mode: on
-        * Running on http://127.0.0.1:3000/ (Press CTRL+C to quit)
-        * Restarting with stat
+5. Fork and clone down the repository
+6. Change into the directory `$ cd eth_crowdfund_be/eth_crowdfund_api`
+7. You will need to work in a virtual environment. Why? Using a virtual environment for Python projects allows us to have an isolated working copy of Python so we can work on a specific project without worrying about affecting other projects.
+8. Within the `/eth_crowdfund_api` directory:  
+ a. Run `# pipenv --three` to create the virtual environment  
+ b. Run `$ pipenv shell` to activate the project virtual environment. When you are done working on the project, you should execute `$ exit` to exit the virtual environment  
+ c. Run `$ pipenv install flask flask-sqlalchemy psycopg2 flask-migrate flask-script marshmallow flask-bcrypt pyjwt` to install all dependencies  
```

Comment:
I'm not sure if we will end up using bcrypt or pyjwt, so we may be able to remove